src/main/java/io/lettuce/core/RedisClient.java

```diff
@@ -429,8 +471,8 @@ public static RedisClient create(ClientResources clientResources, RedisURI redis
      * keys and values. The client {@link RedisURI} must contain one or more sentinels.
      *
      * @param codec Use this codec to encode/decode keys and values, must not be {@code null}
-     * @param <K> Key type
-     * @param <V> Value type
+     * @param <K>   Key type
```

Comment:
Please do not reformat the code.

Reply:
My bad. I forgot to switch off my commit hook in intellij. Reverted